wildfly-standalone-centos8/README.md

```diff
@@ -48,16 +46,15 @@ Following is the Architecture:
 
 ![alt text](images/wildfly-arch.png)
 
-To learn more about the WildFly 18.0.0.Final, visit:
-https://docs.wildfly.org/18/
+To learn more visit the [Wildfly Documentation](https://docs.wildfly.org/18/) site.
```

Comment:
The change mentioned in “Template Solution Architecture” was moved under “Solution Architecture”. Please review and let us know whether we should revert this change